lib/deck.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/deck'
+require './lib/card'
+require 'pry'
```

Comment:
pry is a debugging tool. So, for that reason, I'd just require it in your tests. It doesn't hurt to have it here.. but you don't need it.